### PR TITLE
Update to use `workflow.scenario.preparation`

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Update docs and config to reflect newly public `workflow.scenario.preparation`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Use `docker-compose build --no-cache` to force a rebuild of the Docker image.
     VNET_RESOURCE_GROUP="RMI-PROD-EU-VNET-RG"
     VNET_NAME="RMI-PROD-EU-VNET"
     SUBNET_NAME="RMI-SP-PACTA-DEV-VNET"
-    SUBNET_ID=$(az network vnet subnet show --resource-group $VNET_RESOURCE_GROUP --name $SUBNET_NAME --vnet-name $VNET_NAME --query id -o tsv)
+    SUBNET_ID="/subscriptions/feef729b-4584-44af-a0f9-4827075512f9/resourceGroups/RMI-PROD-EU-VNET-RG/providers/Microsoft.Network/virtualNetworks/RMI-PROD-EU-VNET/subnets/RMI-SP-PACTA-DEV-VNET"
```
Comment (cjyetman) [inline README.md:109]: was this intended?

File: docker-compose.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ services:
         read_only: true
       - type: bind
         source: ${HOST_SCENARIO_INPUTS_PATH}
-        target: /mnt/scenario-sources
+        target: /mnt/workflow-scenario-preparation-outputs
```
Comment (cjyetman) [inline docker-compose.yml:25]: this
https://github.com/RMI-PACTA/workflow.data.preparation/blob/8741ed5d6d5e2163cee6f34396362a3763999801/config.yml#L47

will have to be updated to match too

File: README.md (hunk 1 above)
Comment (AlexAxthelm) [reply README.md:109]: Yeah. @jdhoffa revealed that I'm the only one with read permissions on the VNet, so I'm the only one for whom `az vnet subnet show` will work.

This should enable other with Contributor role to start data prep.

File: docker-compose.yml (hunk 2 above)
Comment (AlexAxthelm) [reply docker-compose.yml:25]: Done in [3a362cc](https://github.com/RMI-PACTA/workflow.data.preparation/pull/219/commits/3a362cc4571b316398cbb8b7859979f524493b07)

**Pull request merged**